File: OnJeong_BE/src/main/java/com/a503/onjeong/domain/user/dto/KakaoDto.java
package com.a503.onjeong.domain.user.dto;

import lombok.Data;

@Data
public class KakaoDto {

    @Data
    public static class Token {
        private String access_token;
        private String refresh_token;
    }

    @Data
    public static class UserInfo {
        private Long kakaoId;
        private String nickname;
        private String profileImageUrl;
    }
}

File: OnJeong_FE/app/src/main/java/com/a503/onjeong/domain/login/dto/KakaoDto.kt
package com.a503.onjeong.domain.login.dto


import com.google.gson.annotations.SerializedName

data class KakaoDto(
    val token: Token
) {
    data class Token(
        @SerializedName("access_token") val accessToken: String,
        @SerializedName("refresh_token") val refreshToken: String
    )
}